include/JSystem/JUtility/JUTGamePad.hpp

```diff
@@ -238,7 +261,7 @@ struct JUTGamePad : public JKRDisposer {
 	static u8 mPadAssign[4];
 	static u32 mSuppressPadReset;
 	static s32 sAnalogMode;
-	static u32 mRumbleSupported;
+	static u32 sRumbleSupported;
```

Comment:
I think this is a rogue field left here from copy-pasting the code from tww or something, it's not present in our linker map so should be removed